v2/pkg/passive/passive.go

```diff
@@ -50,6 +78,54 @@ func (a *Agent) EnumerateSubdomainsWithCtx(ctx context.Context, domain string, p
 	return results
 }
 
+func (a *Agent) buildMultiRateLimiter(ctx context.Context, globalRateLimit int, rateLimit *subscraping.CustomRateLimit) (*ratelimit.MultiLimiter, error) {
+	var multiRateLimiter *ratelimit.MultiLimiter
+	var err error
+	for _, source := range a.sources {
+		var rl uint
+		if sourceRateLimit, ok := rateLimit.Custom.Get(strings.ToLower(source.Name())); ok {
+			rl = sourceRateLimitOrDefault(uint(globalRateLimit), sourceRateLimit)
+		}
+
+		if rl > 0 {
+			multiRateLimiter, err = addRateLimiter(ctx, multiRateLimiter, source.Name(), rl, time.Second)
```

Comment:
i think using `time.Second` as fixed unit is probably not good idea. 
dnsdumpster and lot of other sources have a ratelimit of 1 req/ 2 sec and even worse 10 req /min . since count in `uint` and unit is `Second` we can't get ratelimit of 30 req/ min etc